src/oss/confluent.sh

```diff
@@ -620,7 +625,18 @@ service_exists() {
 
 command_exists() {
     local command="${1}"
-    exists "${command}" "commands"
+    exists "${command}" "commands" || exists "${command}" "enterprise_commands"
+}
+
+cli_file_exists(){
+    if [ -e "${confluent_bin}"/"${1}" ] ; then
```

Comment:
`-f` is preferable because if `sr-acl-cli` is a directory `-e` won't catch it. `-f` works both for files and symlinks to files. 

nit: Wrapping the whole string in `"` works too, if you want to save some keystrokes